Aptoide/aptoide-client-v8
# [AN-1697] Similar posts now send Similar type events instead of recom…

What does this PR do?

   This pull-request fixes a bug that when a Similar card was clicked, it was being parsed as recommendation (not a bug) and sending recommendation analytics events (bug) instead of Similar.

Where should the reviewer start?

- [ ] TimelineAnalytics [line 126] -> check if cardType is SIMILAR when similar card is clicked.

How should this be manually tested?

  Open Aptoide v8 > Timeline > Click in get App on any similar card (either check in Charles the card type for a Similar card or click random non-social get app cards until you get a Similar event). Have in mind that Recommendation and Similar cards are 100% identical, so the charles aproach might be easier to test.

What are the relevant tickets?

Tickets related to this pull-request: [AN-1697](https://aptoide.atlassian.net/browse/AN-1697)

Screenshots (if appropriate):
N/A

Questions:

   Is there a blog post? N/A
   Does this add new dependencies which need to be added? No
